projects/TPVFormer/config/tpvformer_8xb1-2x_nus-seg.py

```diff
@@ -0,0 +1,317 @@
+_base_ = ['mmdet3d::_base_/default_runtime.py']
+
+custom_imports = dict(
+    imports=['projects.TPVFormer.tpvformer'], allow_failed_imports=False)
+
+dataset_type = 'NuScenesSegDataset'
+data_root = 'data/nuscenes/'
+data_prefix = dict(
+    pts='samples/LIDAR_TOP',
+    pts_semantic_mask='lidarseg/v1.0-trainval',
+    CAM_FRONT='samples/CAM_FRONT',
+    CAM_FRONT_LEFT='samples/CAM_FRONT_LEFT',
+    CAM_FRONT_RIGHT='samples/CAM_FRONT_RIGHT',
+    CAM_BACK='samples/CAM_BACK',
+    CAM_BACK_RIGHT='samples/CAM_BACK_RIGHT',
+    CAM_BACK_LEFT='samples/CAM_BACK_LEFT')
+
+backend_args = None
+
+train_pipeline = [
+    dict(
+        type='BEVLoadMultiViewImageFromFiles',
+        to_float32=False,
+        color_type='unchanged',
+        num_views=6,
+        backend_args=backend_args),
+    dict(
+        type='LoadPointsFromFile',
+        coord_type='LIDAR',
+        load_dim=5,
+        use_dim=3,
+        backend_args=backend_args),
+    dict(
+        type='LoadAnnotations3D',
+        with_bbox_3d=False,
+        with_label_3d=False,
+        with_seg_3d=True,
+        with_attr_label=False,
+        seg_3d_dtype='np.uint8'),
+    dict(
+        type='MultiViewWrapper',
+        transforms=dict(type='PhotoMetricDistortion3D')),
+    dict(type='SegLabelMapping'),
+    dict(
+        type='Pack3DDetInputs',
+        keys=['img', 'points', 'pts_semantic_mask'],
+        meta_keys=['lidar2img'])
+]
+
+val_pipeline = [
+    dict(
+        type='BEVLoadMultiViewImageFromFiles',
+        to_float32=False,
+        color_type='unchanged',
+        num_views=6,
+        backend_args=backend_args),
+    dict(
+        type='LoadPointsFromFile',
+        coord_type='LIDAR',
+        load_dim=5,
+        use_dim=3,
+        backend_args=backend_args),
+    dict(
+        type='LoadAnnotations3D',
+        with_bbox_3d=False,
+        with_label_3d=False,
+        with_seg_3d=True,
+        with_attr_label=False,
+        seg_3d_dtype='np.uint8'),
+    dict(type='SegLabelMapping'),
+    dict(
+        type='Pack3DDetInputs',
+        keys=['img', 'points', 'pts_semantic_mask'],
+        meta_keys=['lidar2img'])
```

Comment:
Better remove this line and use the default `meta_keys` in the `__init__`.

Reply:
We only use `lidar2img` in TPVFormer. It will work fine.